src/parser/jobs/mch/modules/Wildfire.tsx

```diff
@@ -0,0 +1,204 @@
+import {t} from '@lingui/macro'
+import {Trans} from '@lingui/react'
+import {ActionLink} from 'components/ui/DbLink'
+import {RotationEvent} from 'components/ui/Rotation'
+import {RotationTable} from 'components/ui/RotationTable'
+import {Event, Events} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {EventHook} from 'parser/core/Dispatcher'
+import {filter} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {History, HistoryEntry} from 'parser/core/modules/ActionWindow/History'
+import {Data} from 'parser/core/modules/Data'
+import Suggestions, {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
+import {Timeline} from 'parser/core/modules/Timeline'
+import React, {Fragment} from 'react'
+import {Message} from 'semantic-ui-react'
+
+// We always want 6 GCDs in WF
+const EXPECTED_GCDS = 6
+
+const SEVERITIES = {
+	BAD_WILDFIRE: {
+		1: SEVERITY.MINOR,
```

Comment:
Not a math nerd so take this with a pinch of salt, but I would personally think the tiers should be more like `MEDIUM` for 1 and `MAJOR` for 3. WF is a pretty important part of the 2-minute burst cycle, so screwing up even one feels like it should be more than `MINOR`.

Reply:
I'm okay with minor here only because it's very easy to miss a single GCD in WF if you have high ping or just needed to clip a second wind or something.